gatsby-config.js

```diff
@@ -2,6 +2,10 @@ const path = require('path');
 const { global } = require('@storybook/design-system');
 const siteMetadata = require('./site-metadata');
 
+require('dotenv').config({
+  path: `.env.${process.env.NODE_ENV}`,
+});
```

Comment:
Env vars aren't available here unless you require them this way. Gatsby suggested doing this.